x-pack/plugins/alerting/server/alerts_service/lib/data_stream_adapter.ts

```diff
@@ -220,9 +223,10 @@ async function createAliasStream(opts: CreateConcreteWriteIndexOpts): Promise<vo
           { logger }
         );
         if (!existingIndices[indexPatterns.name]?.aliases?.[indexPatterns.alias]?.is_write_index) {
-          throw Error(
+          logger.error(
```

Comment:
This one is a little confusing to me.   I think we fall in this code when a concrete write index doesn't exist, so we create one.  We check in the code if it already got created - which is highly likely at startup with multiple Kibanas - and if we find it, but it's not the write index, we fix it so it is.  

I'd think we could depend on whichever other Kibana created the index to make it the write index, but not positive.  So not clear to me we need this here.

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/184161/commits/9096c2f1ac01403f8c3521322a6725acf8331041

I removed the change here to set the write index. It should just throw the error like before.